### PR TITLE
Chore/wjp7 u80z/42 add database yml to gitignore

### PR DESCRIPTION
**Pull request by alexwayne**
pls check. Adding database.yml to gitignore.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,3 +18,6 @@
 
 /notes.txt
 *.patch
+
+# Ignore the database.yml 
+database.yml
```
Comment (gatopan) [inline .gitignore]: Please make sure to configure your editor to add an EOL on files. Won't tolerate this in the future.

File: config/database.yml [hunk 2]
```diff
@@ -1,19 +0,0 @@
...
-
-test:
-  <<: *default
-  host: localhost
-  database: kribi_test
-
-production:
-  <<: *default
-  host: localhost
-  database: kribi_production
```
Comment (gatopan) [inline config/database.yml]: Cool!, It'd be nicer if you left an example database yml for further reference.

File: .gitignore (hunk 1 above)
Comment (alexwayne) [inline .gitignore]: It's a NL, not EOL.

**Comment by gatopan** [comment]
:shipit:

**Pull request merged**